### ISSUE TITLE
Psych sheet

### ISSUE THREAD

**Issue by jfly** [jfly/gjcomps#117]
This is a really neat feature that cubingusa supports, we should definitely support it eventually. Doesn't block Phase 1 though. This is blocked by #116 and #54 .

**Comment by jbcm627** [comment]
I think I had an idea for this... I'm not sure if it was in another ticket or not.  Actually a couple ideas.

1) Re-use navbar real estate for this; can we re-use results tabs, or do we want to keep this separate?
2) How about showing fake times for competitors in rounds that do not yet have results?  (This would address point 1, but I don't know if I like it.)

**Comment by jfly** [comment]
2. Is pretty clever, but I think I'd rather have an explicit "Psych sheet" tab, perhaps nestled under the Registration tab.

**Comment by coder13** [comment]
Irrelevant now?

**Comment by jbcm627** [comment]
This is mostly done by the WCA website. But this may still be interesting to do:

> 2) How about showing fake times for competitors in rounds that do not yet have results? (This would address point 1, but I don't know if I like it.)

**Comment by coder13** [comment]
We can add a toggle to show "projected results".

**Comment by jbcm627** [comment]
Oh, I like that idea.

I do like that it doubles as a psych sheet too - maybe it isn't enough that the WCA has psych sheets, because the registration data will never end up being 100% due to cancellations/additions.

**Comment by jfly** [comment]
I do really like the idea of having projected results. I wonder if we could design a UI for it that makes it super clear what's going on, and wouldn't require us to implement a toggle, maybe just by making the projected results a lightweight gray.
